apps/core-angular-cli/src/app/app.component.html

```diff
@@ -18,27 +18,27 @@
 
   <section cds-layout="grid cols:6 gap:lg">
     <cds-accordion>
-      <cds-accordion-panel [expanded]="panel1Expanded" (expandedChange)="expandedChange($event)">
+      <cds-accordion-section [expanded]="panel1Expanded" (expandedChange)="expandedChange($event)">
         <cds-accordion-header>Item 1</cds-accordion-header>
         <cds-accordion-content *ngIf="panel1Expanded">
           <section>Content 1</section>
         </cds-accordion-content>
-      </cds-accordion-panel>
-      <cds-accordion-panel>
+      </cds-accordion-section>
+      <cds-accordion-section>
```

Comment:
Is this not a breaking change?

Reply:
I think this is a breaking change. Accordion with a few others were moved out of beta in march. This would be good for 6.0

Reply:
Yes, this is a breaking change. Makes sense to go into 6.0. 👍